service/src/test/java/com/ge/predix/controller/test/PolicyEvalWithGraphDbControllerIT.java

```diff
@@ -97,9 +113,19 @@ public void setup() throws Exception {
         MockSecurityContext.mockSecurityContext(this.testZone1);
         MockAcsRequestContext.mockAcsRequestContext(this.testZone1);
         this.policySet = this.jsonUtils.deserializeFromFile("complete-sample-policy-set-2.json", PolicySet.class);
+        TRAVERSAL_LIMIT = graphSubjectRepository.getTraversalLimit();
+        graphSubjectRepository.setTraversalLimit(TEST_TRAVERSAL_LIMIT);
+        graphResourceRepository.setTraversalLimit(TEST_TRAVERSAL_LIMIT);
         Assert.assertNotNull(this.policySet, "complete-sample-policy-set-2.json file not found or invalid");
     }
 
+    @AfterClass
```

Comment:
Instead of setting traversalLimit at the beginning of the whole class and then resetting or cleaning-up after the entire class of tests are done, we can simply do this within the single test method we are writing for evaluating traversal-limit message. 

This can be done within the test method itself. The added advantage of doing so is that now these two [blobs](https://github.com/predix/acs/blob/2c0d68993678945a4e5e62b808734f849e1c0055/service/src/test/java/com/ge/predix/acs/testutils/XFiles.java#L133-L181) become unnecessary. Creating four and five level hierarchy for testing traversal limit is a bit excessive. 


This can also